src/node/README.md

```diff
@@ -19,4 +19,4 @@ The [`src/node/`](./) directory is a new directory introduced in
 [#14978](https://github.com/bitcoin/bitcoin/pull/14978) and at the moment is
 sparsely populated. Eventually more substantial files like
 [`src/validation.cpp`](../validation.cpp) and
-[`src/txmempool.cpp`](../txmempool.cpp) might be moved there.
+[`src/txmempool.cpp`](../txmempool.cpp) may be moved there.
```

Comment:
small NIT 

feel free to ignore this, don't want this to get caught up on this word choice

While `may` can be used in a scenario signaling possibility, it is primarily used in the context of signaling permission to do something. Considering the context, I believe that `might` is the correct choice here.

Reply:
I prefer "may" and disagree with the comment that it is primarily about permission instead of possibiity; both are auxiliary definitions of the word and it's clear that the context here is about possibility.

Reply:
@jonatack you're right